test-triangle/test-triangle.py

```diff
@@ -0,0 +1,78 @@
+import sys
```

Comment:
Your Pull Request title: TRIANGLE-1021 Added test-triangle.py

Check result:

* Pull Request title must start with prefix in ['GENERATOR', 'LEETCODE']